src/resources/InsightPanelConfigs/InsightPanelConfig.ts

```diff
@@ -0,0 +1,41 @@
+import API from '../../APICore';
+import {New, PageModel} from '../BaseInterfaces';
+import Resource from '../Resource';
+import {
+    InsightPanelConfigCreationParams,
+    InsightPanelConfigListOptions,
+    InsightPanelConfigModel,
+} from './InsightPanelConfigInterfaces';
+
+export default class InsightPanelConfig extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/insightconfig/v1/configs`;
+
+    list(options?: InsightPanelConfigListOptions) {
+        const effectiveOptions = {
+            ...options,
+            order: options?.order ?? 'asc',
```

Comment:
Mh...
It's not really the job of the `platform-client` to ensure value defaulting, it's the backend/service job.

Reply:
Yeah I understand and share the feeling, however, the `client` is really 'a dumb piece of code ™️'.
By that I mean its sole job is to 'typescriptify' our REST API, including their quirks.

After that, there are two avenues for addressing the quirks.
My favorite is to "just remove the quirks": change the REST API behavior so that the quirks ain't no more.
However, this is not always possible, an API sometimes needs to be quirky for 90% of the use cases to accommodate 100% of the use cases.

This is where an SDK can help, this is what we do with the `push-api-client.js`: the APIs are complex, but it kinda makes sense for them to be. So, the `platform-client` will wrap them, with their complexity. Then, the SDK (here the `push-api-client.js`) will use the `platform-client` to create 'higher-order-functions' that will wrap the inherent complexity of the API.

In short and sweet:
```mermaid
graph TD;
REST-->platform-client
platform-client-->sdk
```

And you should avoid behavioral change between the REST and platform-client.

Reply:
As you mentioned, removing the default sorting value makes more sense. I'll update that.

Also, it is important to mention that the access to the `insightPanelConfig` resource is currently limited. The default sorting might still change on the service side.